packages/expo-camera/src/ExpoCamera.web.tsx

```diff
@@ -48,17 +44,17 @@ const ExponentCamera = forwardRef(
 
     const isQRScannerEnabled = useMemo<boolean>(() => {
       return Boolean(
-        props.barCodeScannerSettings?.barCodeTypes?.includes('qr') && !!props.onBarCodeScanned
+        props.barcodeScannerSettings?.barcodeTypes?.includes('qr') && !!props.onBarcodeScanned
       );
-    }, [props.barCodeScannerSettings?.barCodeTypes, props.onBarCodeScanned]);
+    }, [props.barcodeScannerSettings?.barcodeTypes, props.onBarcodeScanned]);
 
     useWebQRScanner(video, {
-      interval: props.barCodeScannerSettings?.interval,
+      interval: 300,
```

Comment:
I think we should use the prop here. Hardcoding it to 300 doesn't look right.

Reply:
Hi @lukmccall! 
Thank you for your comment! :D Glad I can be able to contribute.
I do agree, but decided to remove it as;

- the interval prop is not covered in the docs
- the interval prop is not covered in the new type declarations

The `interval` prop does exist on the legacy camera.
Thus, I wrote this text in my PR.

```
The interval property in the BarcodeSettings seems to have been removed in the new version. However, I could not get rid of the interval all together as both the new camera and the legacy uses the same version of useWebQRScanner. For the new camera I set the interval to 300 - this may be something that you need to evaluate.
```

I'm not sure why it was decided it should be removed, but I decided to accept the fact that it's no longer supported and thus added a hard coded value (which may need to be revised).

Reply:
I believe the legacy version of expo-camera should be removed in SDK 52 - if I don't misremember the info from the blog posts. With that change, I believe some (of at least the web stuff) of this package will receive some further attention. 

This PR will currently solve a few issues apart from the reported main issue (e.g. casing of prop names etc). If the `interval`-prop should be reintroduced, perhaps it's better to open a specific PR for it where docs (- and every other affected part) are touched accordingly)?

Reply:
@lukmccall, I did not change the `interval` (see the other posts above) but saw you approved of the changes. I rebuilt the project as this was feedback I received from the workflows. 
To re-run the workflows it need a maintainer approval. Is that something you can help me with?